server/controllers/CommentController.js

```diff
@@ -238,6 +238,50 @@ class CommentController {
       comment,
     });
   }
+
+  /**
+  * @description This method deletes a comment on an article
+  * @param {object} req - exporess request object
+  * @param {object} res - express response object
+  * @param {object} next - express next object
+  * @returns {object} returns an object of comment
+  */
+  static async deleteComment(req, res, next) {
+    const userId = req.userData.id;
+    const { commentId } = req.params;
+
+    let comment;
+    try {
+      comment = await Comment.findByPk(commentId, {
+        raw: true,
+      });
+
+      if (!comment) {
+        return res.status(404).json({
+          status: 'failure',
+          message: 'comment not found'
+        });
+      }
+
+      if (comment.userId !== userId) {
+        return res.status(403).json({
+          status: 'failure',
+          message: 'You cannot delete this comment'
```

Comment:
This error message can be a bit more descriptive, letting the user know why they cannot delete the comment. Something like 'You can only delete your own comments.